api/src/pcapi/core/users/constants.py

```diff
@@ -40,17 +40,39 @@ def __str__(self) -> str:
 
     # If you add a new reason, update `suspend_account()` to cancel
     # bookings if applicable.
-    END_OF_CONTRACT = "end of contract"
-    END_OF_ELIGIBILITY = "end of eligibility"
-    FRAUD = "fraud"
-    UPON_USER_REQUEST = "upon user request"
+    END_OF_CONTRACT = "End of contract"
+    END_OF_ELIGIBILITY = "End of eligibility"
+    FRAUD = "Fraud suspicion"
+    FRAUD_FAKE_DOCUMENT = "Fake document fraud"
+    FRAUD_RESELL_PRODUCT = "Product resell fraud"
+    FRAUD_RESELL_PASS = "Pass resell fraud"
+    FRAUD_DUPLICATE = "Duplicate fraud"
+    FRAUD_HACK = "Hacking fraud"
+    FRAUD_CANCEL = "Cancel booking fraud"
```

Comment:
(style, question de goût) :

```suggestion
    FRAUD_BOOKING_CANCEL = "Booking cancel fraud"
```

plus clair ?

Reply:
oui